examples/aws/terraform/starter-cluster/cluster_sg.tf

```diff
@@ -1,82 +1,100 @@
 /* 
 Security Groups and Rules for Cluster.
 
-Note: Please see the list of networking ports documentation for their usage. 
+Note: Please see the list of networking ports documentation for their usage.
 https://goteleport.com/docs/setup/reference/networking/#ports
 */
 
 // Create a Security Group
 resource "aws_security_group" "cluster" {
-  name   = "${var.cluster_name}-cluster"
-  vpc_id = data.aws_vpc.default.id
+  name        = "${var.cluster_name}-cluster"
+  description = "${var.cluster_name} cluster"
+  vpc_id      = data.aws_vpc.default.id
 
   tags = {
     TeleportCluster = var.cluster_name
   }
 }
 
 // Permit inbound to SSH
+// tfsec:ignore:aws-ec2-no-public-ingress-sgr
 resource "aws_security_group_rule" "cluster_ingress_ssh" {
+  description       = "Permit inbound to SSH"
   type              = "ingress"
   from_port         = 22
   to_port           = 22
   protocol          = "tcp"
   cidr_blocks       = var.allowed_ssh_ingress_cidr_blocks
   security_group_id = aws_security_group.cluster.id
 }
+
 // Permit inbound to Teleport Web interface
+// tfsec:ignore:aws-ec2-no-public-ingress-sgr
 resource "aws_security_group_rule" "cluster_ingress_web" {
+  description       = "Permit inbound to Teleport Web interface"
   type              = "ingress"
   from_port         = 3080
   to_port           = 3080
   protocol          = "tcp"
   cidr_blocks       = var.allowed_ingress_cidr_blocks
   security_group_id = aws_security_group.cluster.id
 }
+
 // Permit inbound to Teleport services
+// tfsec:ignore:aws-ec2-no-public-ingress-sgr
 resource "aws_security_group_rule" "cluster_ingress_services" {
+  description       = "Permit inbound to Teleport services"
   type              = "ingress"
   from_port         = 3022
   to_port           = 3025
   protocol          = "tcp"
   cidr_blocks       = var.allowed_ingress_cidr_blocks
   security_group_id = aws_security_group.cluster.id
 }
+
 // Permit inbound to Teleport mysql services
+// tfsec:ignore:aws-ec2-no-public-ingress-sgr
 resource "aws_security_group_rule" "cluster_ingress_mysql" {
+  description       = "Permit inbound to Teleport mysql services"
   type              = "ingress"
   from_port         = 3036
   to_port           = 3036
   protocol          = "tcp"
   cidr_blocks       = var.allowed_ingress_cidr_blocks
   security_group_id = aws_security_group.cluster.id
-  count  = var.enable_mysql_listener ? 1 : 0
+  count             = var.enable_mysql_listener ? 1 : 0
```

Comment:
I don't quite understand why some of these changes in the diff are removing extra tabs while some others are adding them in. Automatic rather than deliberate?

Reply:
Yeah, that's from `terraform fmt`. terraform has its own preferred style that it likes to enforce (and the linting jobs enforce it as well).